### ISSUE TITLE
ML: Pretrained Model #1 (MNIST): Configuration#1: Training model and store the results in the directory

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#256]
1. Put the training on the Grid by using jetscape-ml-tensorflow-nn-mnist.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
840000/840000 [==============================] - 9s 11us/sample - loss: 0.6281 - accuracy: 0.7198 - val_loss: 0.6261 - val_accuracy: 0.7387
{'history': {'loss': [0.6928050813402449, 0.6921829956145513, 0.6915314816066197, 0.6907486990519932, 0.689893963268825, 0.688991111664545, 0.6880686896823701, 0.6871390720004127, 0.68623297391619, 0.6853697842098417, 0.6844758614585513, 0.6836192219234648, 0.6827952413922265, 0.6819401469639369, 0.6811035045669193, 0.6802047783261254, 0.6793577071553185, 0.6784010311580839, 0.6774704520270938, 0.6764575861067998, 0.6754203874497187, 0.6743783796764555, 0.6732432701065427, 0.67213215977805, 0.6709033710252671, 0.6697304762567793, 0.6683846409661429, 0.667103459085737, 0.665680156993866, 0.6642633377710978, 0.6627749471755254, 0.6613541648864746, 0.659828744552249, 0.6582892659550621, 0.656548859782446, 0.6549413497879392, 0.6531790706452869, 0.6514744208653768, 0.6497317297890073, 0.6478978196734474, 0.6460324457168579, 0.6441157747813634, 0.6422127149990626, 0.6402540658315022, 0.638270450873602, 0.6363504468781608, 0.6342856246221633, 0.6321400551568894, 0.6301479230517433, 0.6280555476688203], 'accuracy': [0.50558573, 0.5263226, 0.5432786, 0.55352616, 0.5602119, 0.5627667, 0.5655607, 0.5667964, 0.5691417, 0.5703762, 0.5720107, 0.57429284, 0.5767643, 0.5782595, 0.5809631, 0.5837345, 0.586425, 0.5892976, 0.5925976, 0.5958167, 0.59927976, 0.60332024, 0.6067226, 0.61024404, 0.61468333, 0.61845595, 0.6231821, 0.62704164, 0.63209164, 0.6361893, 0.64100236, 0.64512146, 0.64991426, 0.6537988, 0.6589619, 0.66340595, 0.66838217, 0.6724845, 0.6767726, 0.68101907, 0.68514764, 0.68983096, 0.6936024, 0.6978452, 0.7016405, 0.7056774, 0.7089595, 0.7132274, 0.7168857, 0.71980953], 'val_loss': [0.6924359085718791, 0.6918128955523173, 0.6911102340698242, 0.690308921400706, 0.6894334361712138, 0.6885205823898315, 0.6875999239285787, 0.6866996867815653, 0.6858184477806092, 0.6849509423573812, 0.6841005738576253, 0.6832612746238709, 0.682422765159607, 0.681576844851176, 0.6807200223286947, 0.6798286178906758, 0.6789283727010091, 0.6779940371831258, 0.6770241949717204, 0.6760163071314493, 0.6749666593551635, 0.6738776617685954, 0.6727361570994059, 0.671556564617157, 0.6703334395090739, 0.6690685352961222, 0.6677321141878764, 0.6663820395151774, 0.6649445883750915, 0.6635016746520996, 0.6619879820505779, 0.6604354266484579, 0.6588479641596476, 0.6572127110799154, 0.655527081712087, 0.6537981576919556, 0.6520362052281697, 0.6502398225148519, 0.6484173187891642, 0.6465304272333781, 0.6446276492436727, 0.6426992577552796, 0.640726549466451, 0.6387300637563069, 0.6366800852775574, 0.634644294389089, 0.632535684967041, 0.6304129671414693, 0.6282725474675497, 0.626118101755778], 'val_accuracy': [0.5191, 0.53955, 0.55328333, 0.5611042, 0.56349164, 0.56577915, 0.5678375, 0.56957084, 0.5712583, 0.57310414, 0.57524586, 0.57722086, 0.5794542, 0.5822333, 0.5851, 0.587625, 0.5909208, 0.5941125, 0.5973333, 0.6009333, 0.60472083, 0.60847086, 0.61240417, 0.6165958, 0.6212958, 0.6263458, 0.63044167, 0.6359375, 0.6394375, 0.64575833, 0.65073335, 0.6560417, 0.66095835, 0.66622084, 0.6714542, 0.6762583, 0.68160415, 0.6865917, 0.691525, 0.69635415, 0.7009708, 0.70516664, 0.70965415, 0.7141125, 0.7188542, 0.72269166, 0.7269625, 0.7311958, 0.73495, 0.7386917], 'lr': [5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06, 5e-06]}}

**Comment by hmehryar** [comment]
inserted into Excel to port into overleaf

**Comment by hmehryar** [comment]
inserted into Excel to port into overleaf
